docs/src/main/sphinx/admin/properties-general.md

```diff
@@ -67,3 +67,24 @@ size limits.
 Prepared statement compression is not applied if the size gain is less than the
 configured value. Smaller statements do not benefit from compression, and are
 left uncompressed.
+
+(file-compression)=
+## File compression and decompression
+
+Trino uses the [aircompressor](https://github.com/airlift/aircompressor) library
+to compress and decompress ORC, Parquet, and other files using the LZ4, zstd,
+Snappy, and other algorithms. The library takes advantage of using embedded,
+higher performing, native implementations for these algorithms by default. 
+
+If necessary, this behavior can be deactivated to fall back on JVM-based
+implementations with the following configuration:
+
+```properties
+io.airlift.compress.v3.disable-native=true
```

Comment:
This needs to go in the jvm.config file, not the server config properties.

Reply:
oh ..

Reply:
Updated

Reply:
fixed